docs/BicepInstallation.md

```diff
@@ -0,0 +1,104 @@
+# Installation options via Bicep Templates for use with FHIR service in Azure Health Data Services 
+This article details three deployment options for provisioning and installation of the IoMT FHIR Connector for Azure and [FHIR service in Azure Health Data Services](https://learn.microsoft.com/en-us/azure/healthcare-apis/fhir/overview) using Bicep templates. 
+
+The following Azure components will be provisioned once deployment has completed:
+
+* Storage Account 
+* Event Hubs Namespace  
+* Two Event Hubs 
+* Azure Health Data Services Workspace
+* FHIR Service* 
+* Azure Container Registry 
+* Log Analytics Workspace 
+* App Insights 
+* Managed Identity 
+* Container Apps Environment
+* Two Container Apps  
+
+An additional Storage Account and Container Instances will be created if the 'deploymentScripts' Bicep resource type is used in the chosen deployment option (Options 1 and 2). 
+
+\* NOTE: These deployment options use the FHIR service in Azure Health Data Services as opposed to the Azure API for FHIR. For more information on the differences between these services, please see this [page](https://learn.microsoft.com/en-us/azure/healthcare-apis/fhir/overview). 
+
+### Prerequisites
+To run any of these deployment options, the following items must be set up before execution:
+
+* Contributor and User Access Administrator OR Owner permissions on your Azure subscription 
```

Comment:
I feel that we don't have to mention this as prerequisite because customers may have their own way to manage and grant the permission granularity for their subscription.

Reply:
Thanks for pointing this out, I didn't consider that customers may have different permission granularities. Out of curiosity, are most OSS users familiar with Azure and the permissions needed to perform these deployments? I put this as a requirement since whoever is running it will need permissions to deploy resources and assign role assignments in their subscription, otherwise they will get an error when they go to deploy. Do you think there is a better way to call this out or do you think it's not necessary?

Reply:
I do believe it is worth calling out they need at least Contributor and User Access Admin to run the scripts.  These are standard Azure roles and their permissions are well known.